<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="startPage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../rc/images.qrc">:/ownimages/mimagesCC/Maracana_1.jpg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="newLocalGamePB">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>20</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>100</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>New game</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+N</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="openGamePushButton">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Open game</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+L</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="quitPushButton">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Quit</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+Q</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="NewGamePage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&amp;Country:</string>
          </property>
          <property name="buddy">
           <cstring>countryCB</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="countryCB">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="7">
         <widget class="QListWidget" name="teamsList"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>&amp;League:</string>
          </property>
          <property name="buddy">
           <cstring>leagueCB</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="leagueCB">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>43</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Player &amp;Name:</string>
          </property>
          <property name="buddy">
           <cstring>gamesterLE</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="gamesterLE">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&amp;Start League:</string>
          </property>
          <property name="buddy">
           <cstring>startLeagueCB</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="startLeagueCB">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="addGamesterPushButton">
          <property name="text">
           <string>Add Player</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Connected Player:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="maxGamestersLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="3">
         <widget class="QTableWidget" name="gamesterTableWidget">
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <stylestrategy>PreferDefault</stylestrategy>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Country</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Team</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>start League</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="8" column="0" colspan="3">
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="backPushButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeGamesterPushButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Remove player</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="startGamePushButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Start game</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../rc/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>setStackedWidgetIndex(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newLocalGamePB</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>newLocalGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>setStackedWidgetIndex(int)</signal>
  <slot>newLocalGame()</slot>
  <slot>startPage()</slot>
 </slots>
</ui>
